web/download_manager.js

```diff
@@ -86,20 +86,20 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC || CHROME')) {
     },
 
     download: function DownloadManager_download(blob, url, filename) {
-      if (!URL) {
```

Comment:
I just tested it out on IE11. This change makes IE11 use the `downloadUrl` fallback rather than `navigator.msSaveBlob`.

I could change the condition to:

```js
if (!URL || pdfjsLib.PDFJS.disableCreateObjectURL && !navigator.msSaveBlob)
```

if that's clearer than swapping the order of the two `if` blocks?